src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -195,8 +187,45 @@ internal static void RegisterOtlpExporterOptionsFactory(IServiceCollection servi
         string name)
         => new(
             configuration,
+            OtlpExporterOptionsConfigurationType.Default,
             serviceProvider.GetRequiredService<IOptionsMonitor<BatchExportActivityProcessorOptions>>().Get(name));
 
+    internal void ApplyConfigurationUsingSpecificationEnvVars(
+        IConfiguration configuration,
+        string endpointEnvVarKey,
+        bool appendSignalPathToEndpoint,
+        string protocolEnvVarKey,
+        string headersEnvVarKey,
+        string timeoutEnvVarKey)
+    {
+        if (configuration.TryGetUriValue(endpointEnvVarKey, out var endpoint))
+        {
+            this.endpoint = endpoint;
+            if (!appendSignalPathToEndpoint)
```

Comment:
Could we simplify this to `this.AppendSignalPathToEndpoint = appendSignalPathToEndpoint;`?